laser_slam_tools/src/octomap_to_point_cloud.cc

```diff
@@ -0,0 +1,48 @@
+#include <glog/logging.h>
```

Comment:
Should I always use .cpp files when contributing to your repositories? The contribution section in the segmatcher wiki says you adhere to the google C++ style guide which uses .cc files.